components/playground/playground.go

```diff
@@ -431,6 +433,58 @@ func (p *Playground) bootCluster(options *bootOptions) error {
 				return
 			}
 		}()
+
+		// set up grafana
+		if err := installIfMissing(p.profile, "grafana", options.version); err != nil {
```

Comment:
Can we put all `installIfMissing` at the beginning of the start playground and install them concurrently?

Reply:
sounds good, can you fire an issue?

Reply:
https://github.com/pingcap/tiup/issues/491